spec/index.bs

```diff
@@ -6727,7 +6746,16 @@ attachments used by this encoder.
 
             **Returns:** {{undefined}}
 
-            Issue: Describe {{GPURenderPassEncoder/endOcclusionQuery()}} algorithm steps.
+            Issue the following steps on the [=Device timeline=] of |this|.{{GPUObjectBase/[[device]]}}:
+            <div class=device-timeline>
+                1. If any of the following conditions are unsatisfied,  generate a validation error and stop.
+                    <div class=validusage>
+                        - |this|.{{GPURenderPassEncoder/[[occlusion_query_set]]}} is [$valid to use with$] |this|.{{GPUObjectBase/[[device]]}}.
+                        - The query can not be ended if it is not begun.
```

Comment:
This could just check the boolean (if it is set then it means the occlusion query set is not null and valid to use with the device.)